plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -1673,22 +1673,17 @@ public Optional<ConnectorTableExecuteHandle> getTableHandleForExecute(
             Map<String, Object> executeProperties,
             RetryMode retryMode)
     {
-        DeltaLakeTableHandle tableHandle = (DeltaLakeTableHandle) connectorTableHandle;
-
-        DeltaLakeTableProcedureId procedureId;
```

Comment:
Keep this part as it was before.
There can appear IAE while initializing the `procedureId`.

Reply:
reverted